esy-install/DistStorage.re

```diff
@@ -137,6 +174,7 @@ let fetch' = (sandbox, dist, gitUsername, gitPassword) => {
         let%bind () = Git.checkout(~ref=github.commit, ~repo=stagePath, ());
         let%bind () = Git.updateSubmodules(~config, ~repo=stagePath, ());
         let%bind () = Fs.rename(~skipIfExists=true, ~src=stagePath, path);
+        // TODO: handle extraSouces for Git repos
```

Comment:
Shall we track this in a separate issue?

Reply:
Yes that would be better I feel, as I did not encounter an immediate need for extra-sources for git repos